Add blazor template, update all to 8.0 (#38662)
dotnet/docs: docs/core/tools/dotnet-new-sdk-templates.md

***

## `blazor`

- **`-f|--framework <FRAMEWORK>`**

  Specifies the [framework](../../standard/frameworks.md) to target.

  This template is available for .NET 8 or later.

- **`--no-restore`**

  Doesn't execute an implicit restore during project creation.

- **`--exclude-launch-settings`**

  Excludes *launchSettings.json* from the generated app.

- **`-int|--interactivity <None|Server|Webassembly|Auto >`**

>   Which interactive render mode to use for interactive components. The possible values are:

Reviewer: ```suggestion
  Specifies which interactive render mode to use for interactive components. The possible values are:
```

  - `None` - No interactivity (static server rendering only)

>   - `Server` - (Default) Runs on the server

Reviewer 2: ```suggestion
  - `Server` - (Default) Runs the app on the server with interactive server-side rendering.
```

>   - `WebAssembly` - Runs in the browser using WebAssembly

Reviewer 2: ```suggestion
  - `WebAssembly` - Runs the app using client-side rendering in the browser with WebAssembly.
```

>   - `Auto` -Uses Server while downloading WebAssembly assets, then uses WebAssembly

Reviewer 2: ```suggestion
  - `Auto` - Uses interactive server-side rendering while downloading the Blazor bundle and activating the Blazor runtime on the client, then uses client-side rendering with WebAssembly.
```
